Runtime/UI/Utils/SpriteStretch.cs

```diff
@@ -68,12 +58,22 @@ public void StretchSprite()
             gameObject.transform.localScale = new Vector3(scaleFactorX, scaleFactorY, 1);
         }
 
-        public void CalculateCameraSize()
+        // TODO: separate this in another script and package
+        public static Vector2 GetCameraSize(Camera cameraToGetSize)
```

Comment:
This parameter can be called `camera` only, the name of the method already implies it will be used to get the size from it